pkg/printers/internalversion/printers.go

```diff
@@ -60,7 +59,20 @@ import (
 	"k8s.io/kubernetes/pkg/util/node"
 )
 
-const loadBalancerWidth = 16
+const (
+	loadBalancerWidth = 16
+
+	// LabelNodeRoleMaster specifies that a node is a master
+	// It's copied over to here until it's merged in core: https://github.com/kubernetes/kubernetes/pull/39112
+	LabelNodeRoleMaster = "node-role.kubernetes.io/master"
```

Comment:
Afaik a taint alone cannot guarantee that a pod ends up on a master node. So this is both the label and the taint. 39112 has been agreed, although we probably need the steering committee to get it merged.

As for delaying this further, this fixes a regression in kubectl where it does not identify the master. It is an annoying regression, having hit it in the real world.